### PR TITLE
Add working agreement guide for technical event speakers

### PR DESCRIPTION
**Pull request by wsbctechnicalblog**
Tips and ticks for preparing as a technical event speaker, from preparing your message to engaging your audience effectively..

Preparing for a technical community event, such as the [DevOps Vancouver meetup](https://www.meetup.com/DevOps-Vancouver-BC-Canada), is essential to deliver an impactful message. A well-prepared speaker builds credibility and trust, ensuring the presentation is relevant and engaging. This preparation boosts confidence, allows for smoother delivery, and respects the audience's time.

### CODE REVIEW

File: content/technical-event-speakers.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+> Technical Speaker Tips
+>
+> ![speaker tips](../images/technical-event-speakers-3.png) 
+
+As summarised on the working agreement cheat sheet, we recommend that you:
+
+## Preparation
+
+- **Know your aAudience** - Tailor content to their expertise. Research backgrounds if possible.
```
Comment (sben101) [inline content/technical-event-speakers.md]: Typo for Audience

**Review by sben101 — COMMENTED** [review]
I can use these tips for my own talk that I do in the future!  I have provided my one comment in the 'preparation' section for a typo. Other than that, looks pretty amazing!

**Comment by MerlinDarkstar** [comment]
[heart]         Lacey, Martin reacted to your message:
________________________________

File: content/technical-event-speakers.md (hunk 1 above)
Comment (wpschaub) [reply content/technical-event-speakers.md]: Fixed.

**Review by MerlinDarkstar — APPROVED** [review]

**Review by danielrbroderick — APPROVED** [review]

**Pull request merged**